python/tests/test-export-classify.sh

```diff
@@ -0,0 +1,89 @@
+BASELINE_DIR=${HOME}/dev/work/baseline
```

Comment:
There should probably be a version check for tensorflow here, My tf 1.8 install was failing these scripts.

```bash
python -c 'import tensorflow; from distutils.version import LooseVersion; import sys; i = 1 if LooseVersion(tensorflow.__version__) < LooseVersion("1.12.0") else 0; sys.exit(i)'
```

Then you can check the return value to see if you should abort